src/_components/Footer.jsx

```diff
@@ -11,6 +10,10 @@ export default function Footer() {
   const linkedin_url = "https://www.linkedin.com/company/stevens-blueprint/";
   const github_url = "https://github.com/stevensblueprint";
   const instagram_url = "https://instagram.com/stevensblueprint";
+  const blog_url = "/blog";
+  const support_url = "/support";
+  const opp_url = "/opportunities";
+  const timeline_url = "/fornpos#timeline-section"
```

Comment:
timeline_url would be 'non-profits#timeline-section`. The Navbar also has the correct routes for the base pages. Just as a note, all the routes are defined in the markdown (.md) files of the src directory: 
![image](https://github.com/user-attachments/assets/2e460129-0772-4c9f-965d-e6ad49aff83d)